### ISSUE TITLE
Color of the key words in Rust is inconsistent

### ISSUE THREAD

**Issue by Scott169** [luqimin/tinylight-vscode#11]
Hello! I noticed that the recent versions changed the color of many elements, and I found the colors of some key words were different in newer versions.

I would have a try to fix it by myself and make a pr for it. If you could help fix that would be grateful.

![image](https://github.com/luqimin/tinylight-vscode/assets/38151815/dbd7a9c2-86a5-49a6-bfda-16b0f64affbf)

**Comment by luqimin** [comment]
Yes, of cource! Sorry for the late reply.

**Comment by Scott169** [comment]
Thanks, but there's not change😂

By the way, there's inconsistent of keywords in rendered documentation (color, bold). All these issues are only found in v2.0.0 and later.

![image](https://github.com/luqimin/tinylight-vscode/assets/38151815/f613b632-f12e-4b9a-a67b-cb6e68fdc629)

**Comment by luqimin** [comment]
> Thanks, but there's not change😂
> 
> By the way, there's inconsistent of keywords in rendered documentation (color, bold). All these issues are only found in v2.0.0 and later.
> 
> ![image](https://private-user-images.githubusercontent.com/38151815/331335603-f613b632-f12e-4b9a-a67b-cb6e68fdc629.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYwMDM1NTAsIm5iZiI6MTcxNjAwMzI1MCwicGF0aCI6Ii8zODE1MTgxNS8zMzEzMzU2MDMtZjYxM2I2MzItZjEyZS00YjlhLWE2N2ItY2I2ZTY4ZmRjNjI5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE4VDAzMzQxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxNjJiYmJkNWU4Yjc2ZTQxY2VkY2Y3MmFhZWYxMjZhMGY1MGMxNjJhYzVlNWNhNTZlODgzNTVkNmYxZmE4MGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.23D0_QUGL5mog1XggZ8dGyLv0Fa9QO3LUIYyKLtIWKU)

Can you please give me a code demo, so I can debug directly.
Of course, it would be great if you can make a pr 😁

**Comment by Scott169** [comment]
How to provide the code demo to you?

I can help debug this issue by comparing v2.0.6 and v0.10.1 to find which elements were changed.

**Comment by luqimin** [comment]
In my workspace, the snippet looks like this, perhaps some plugin breaks the original style?

<img width="649" alt="image" src="https://github.com/luqimin/tinylight-vscode/assets/12484305/fcec10e9-c63b-4c78-87b7-54417d509858">

**Comment by Scott169** [comment]
Yes, I noticed that. Maybe it is caused by [rust-analyzer](https://github.com/rust-lang/rust-analyzer). The color is changed when the indexing is finished, see recorded video below:

https://github.com/luqimin/tinylight-vscode/assets/38151815/dd5f045d-12e7-44cd-bdf8-423b5f4a40e0
